README.md

```diff
@@ -64,4 +64,4 @@ If you have suggestions for how this component could be improved, or want to rep
 For more, check out the [Contributing Guide](CONTRIBUTING.md).
 
 ## License
-[ISC](LICENSE) © 2020 Sebass van Boxel <hello@svboxel.com>
```

Comment:
@olcayertas Why are you updating the npm and license source?

Reply:
I was try to create the pull request for ow fork. I will close this one.

Reply:
Feel free to submit your changes updates upstream. We'll make sure to get you listed as a contributor.